3493/0/about_dice_project.rb

```diff
@@ -0,0 +1,69 @@
+require File.expand_path(File.dirname(__FILE__) + '/neo')
+
+# Implement a DiceSet Class here:
+#:reek:FeatureEnvy:reek:Attribute:
+class DiceSet
+  attr_accessor :values
+  def roll(number)
+    @values = (0...number).map { Random.rand(1..6) }
```

Comment:
You need to set [new instance of array](http://ruby-doc.org/core-2.5.0/Array.html)